src/web/src/components/Posts/Post.tsx

```diff
@@ -16,16 +16,16 @@ import {
 } from '@material-ui/core';
 import ErrorRoundedIcon from '@material-ui/icons/ErrorRounded';
 import LiteYouTubeEmbed from 'react-lite-youtube-embed';
-import GithubInfoProvider from '../GithubInfoProvider';
-import { Post } from '../../interfaces';
-import AdminButtons from '../AdminButtons';
-import Spinner from '../Spinner';
-import PostDesktopInfo from './PostInfo';
-import PostAvatar from './PostAvatar';
-import GitHubInfo from './GitHubInfo';
+import GitHubInfo from './index';
```

Comment:
```suggestion
import GitHubInfo from './GitHubInfo';
```

Reply:
Is this to include all of the files in the `GitHubInfo` file?

Reply:
GitHubInfo folder has an index file so importing from `./GitHubInfo`  will resolve to the index file.

Reply:
ok got it. Thank.